jupyterhub/values.schema.yaml

```diff
@@ -2757,6 +2757,12 @@ properties:
           See [the Kubernetes
           documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
           for more details about annotations.
+      customPaths:
```

Comment:
We've generally named properties that extend an existing object `extra*` rather than `custom*`

Reply:
of course makes sense